Game.cpp

```diff
@@ -190,9 +191,17 @@ void Game::processInput()
                         continue;
                 }
                 lockPiece();
+                return; // don't check arduino
                 //std::cout << evnt.key.keysym.scancode << std::endl;
         }
     }
+    bool lock;
+    lock = curShape->move(pc->moveLeft - (int)pc->moveRight);
+    lock = curShape->rotate(pc->rotateLeft - (int)pc->rotateRight) || lock;
+    lock &= !(pc->fastDrop && pc->instantDrop);
+    toggleFastDrop(pc->fastDrop);
+    if(pc->instantDrop) instantDrop();
+    if(lock) lockPiece();
 }
```

Comment:
this is where the arduino input is interpreted. I redefined move and rotate for shape so that I can input 0 to it and it'll do nothing.